docs/src/main/tut/json.md

```diff
@@ -243,7 +243,7 @@ Finally, we post `User("Alice")` to our Hello service and expect
 `Hello("Alice")` back:
 
 ```tut:book
-helloClient("Alice").run
+//helloClient("Alice").unsafeRun
```

Comment:
You're not it's a bug in tut in 2.12 that requires Tasks to be declared and run on separate lines or else it hangs.